apps/src/lib/cli.rs

```diff
@@ -2342,6 +2342,22 @@ pub mod args {
     );
     pub const BLOCK_HEIGHT: Arg<BlockHeight> = arg("block-height");
     // pub const BLOCK_HEIGHT_OPT: ArgOpt<BlockHeight> = arg_opt("height");
+    pub const BRIDGE_POOL_GAS_AMOUNT: ArgDefault<token::DenominatedAmount> =
```

Comment:
This amount will need to be validated. When reading from string, it gives it the minimum denom possible to fit the amount. We need to look up the denom from storage and increase it if necessary. If too much precision was given, we must fail.